### ISSUE TITLE
Add a skip link to the admin area

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#15339]
**Describe the bug**

Like the client-facing portion of a Forem instance, the admin area requires a skip link as well to make the admin experience more accessible.

**To Reproduce**

1. Navigate to any page in the admin section, e.g. http://localhost:3000/admin
2. Use the TAB key to navigate the site
3. No skip link appears and an admin is forced to navigate through the entire sidebar menu.

**Expected behavior**

A skip link should be included so an admin can access the main region of a page in the admin section.

**Screenshots**

**Desktop (please complete the following information):**

- OS, version:
- Browser, version:

**Smartphone (please complete the following information):**

- Device:
- OS, version:
- Browser, version:

**Additional context**
